src/main/java/com/google/codeu/data/TeamDatastore.java

```diff
@@ -61,4 +65,15 @@ public Team getTeam(Entity teamEntity) {
         String githubLink = (String) teamEntity.getProperty("githubLink");
         return new Team(teamID, cohortID, teamName, projectName, projectDesc, githubLink);
     }
+    
+    public List<Team> getTeams(String cohortId) {
+  	  List<Team> teams = new ArrayList<Team>();
+  	  Query query = new Query("Team")
+  		  .setFilter(new Query.FilterPredicate("cohortId", FilterOperator.EQUAL, UUID.fromString(cohortId)));
```

Comment:
This can just be the cohortId.

```suggestion
  		  .setFilter(new Query.FilterPredicate("cohortId", FilterOperator.EQUAL, cohortId));
```